pom.xml

```diff
@@ -368,24 +370,83 @@
         </dependency>
       </dependencies>
     </profile>
+    
+    <profile>
+      <id>release</id>
+      <properties>
+        <gpg.executable>gpg2</gpg.executable>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.7</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>ossrh</serverId>
+              <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+              <autoReleaseAfterClose>true</autoReleaseAfterClose>
+            </configuration>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-gpg-plugin</artifactId>
+            <version>1.6</version>
+            <executions>
+              <execution>
+                <id>sign-artifacts</id>
+                <phase>verify</phase>
+                <goals>
+                  <goal>sign</goal>
+                </goals>
+                <configuration>
+                  <keyname>${gpg.keyname}</keyname>
+                  <passphraseServerId>${gpg.keyname}</passphraseServerId>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
   </profiles>
 
-<!--   <scm> -->
-<!--     <connection>scm:?????</connection> -->
-<!--     <developerConnection>scm:?????</developerConnection> -->
-<!--     <url>?????</url> -->
-<!--   </scm> -->
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+  
+  <developers>
+    <developer>
+      <name>Graham Crockford</name>

```

Comment:
We actually have a complete developer list!  Should at least have more than one of us here.

Reply:
I've added everyone who has contributed on GitHub based on https://github.com/alfasoftware/morf/graphs/contributors in alphabetical order. Hope I got all the details right :)

Reply:
I think just the key maintainer list is enough.  Otherwise we'll be changing this list constantly.

Reply:
Changed it myself